helpers/constants.js

```diff
@@ -62,7 +62,8 @@ const MESSAGE_TYPE = {
   GIF: 'gif',
   LOCATION: 'location',
   SOUND: 'sound',
-  DELETED: 'deleted'
+  DELETED: 'deleted',
+  TEXT: 'text'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a new message type 'TEXT' to the MESSAGE_TYPE enum is a good idea, considering that text messages are one of the most common types of messages in chat applications. However, please ensure that this new type is handled correctly wherever MESSAGE_TYPE is used throughout the application.
```diff
+  TEXT: 'text'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->